chia/types/coin_spend.py

```diff
@@ -50,7 +50,16 @@ def compute_additions_with_cost(
             raise ValidationError(Err.BLOCK_COST_EXCEEDS_MAX, "compute_additions() for CoinSpend")
         atoms = cond.as_iter()
         op = next(atoms).atom
-        if op in [ConditionOpcode.AGG_SIG_ME, ConditionOpcode.AGG_SIG_UNSAFE]:
+        if op in [
```

Comment:
This doesn't seem to be gated on the hard fork taking effect. Maybe it doesn't matter since these costs are just supposed to be estimates anyway.